### ISSUE TITLE
Corporation Tax: Research and Development tax relief

### ISSUE THREAD

**Issue by nelsonic** [dwyl/hq#112]
We _risk_ making a "Profit" this year because I have left _90%_ my (_contracting_) earnings in the business to ensure we have _capital_ to work on Products. 
Thankfully instead of paying 20% Tax on the money, we can use R&D Tax Relief:
https://www.gov.uk/guidance/corporation-tax-research-and-development-rd-relief

> # NextAction: Call: George on Monday!

**Comment by nelsonic** [comment]
I _tried_ to call George but did not get through to him. Sent him a detailed text message instead. 
Hope to have a bit more luck tomorrow.

**Comment by iteles** [comment]
Meeting this Thursday 👍

**Comment by nelsonic** [comment]
@iteles please bring @markwilliamfirth up-to-speed on this _early_ next week 
(_once all accounts are prepared_) so we can pick this up ***soon***! thanks!

**Comment by iteles** [comment]
George called me last Friday, will call again in late February and we can start the process for 2016/17 then 👍

**Comment by markwilliamfirth** [comment]
[GrantTree ](https://granttree.co.uk/) is another company that can do R&D tax credit reports.

You can claim up to two years.

**Comment by markwilliamfirth** [comment]
Currently in correspondence with GrantTree and Leyton

**Comment by markwilliamfirth** [comment]
@iteles would you be able to give me an estimate of how much development time is spent on the projects with pay contractors to do that we retain the IP for?

i.e. percentage of time contractors are **paid** to work on DWYL modules and in-house projects

**Comment by markwilliamfirth** [comment]
<img width="709" alt="screen shot 2017-02-16 at 17 31 57" src="https://cloud.githubusercontent.com/assets/5732209/23032867/e6e44b08-f46d-11e6-8626-37b7ae550d1f.png">

^question from GrantTree in order to calculate our claim estimate

**Comment by iteles** [comment]
We can calculate how many client projects.

On dev time, does the seniority of the dev matter (i.e. higher day rates or lower)?

**Comment by markwilliamfirth** [comment]
Yes - I think he basically wants an estimate of how much money goes towards our own IP so senior developer rates will make a difference

**Comment by markwilliamfirth** [comment]
@iteles do you have an estimate for this?

**Comment by markwilliamfirth** [comment]
@nelsonic I believe @iteles said you might know the estimate for this? Can someone give me a figure please? 😄

**Comment by nelsonic** [comment]
This is an _easy_ calculation that _anyone_ can derive with some simple arithmetic. ***£400k***.

**2k hours** _per year_ x **2 years** x **£100/hr** (_my "market/going rate"_) 
this is a ***conservative estimate*** of the time I have invested so far in dwyl modules
and I would be _happy_ to explain _exactly_ how I spent that time to someone at HMRC if they ask. 👍 

# tl;dr

[Research and Development Tax Credits](https://www.gov.uk/guidance/corporation-tax-research-and-development-rd-relief) exist for _one_ reason: to support the _investment_ in Technolgoy in British Business which will lead to value/jobs in the UK economy. We are generating value (_as evidenced by how much **VAT** we've paid_) and creating Jobs ask anyone who has received a payment from @dwyl recently. 😉 

_ALL_ the work I have done @dwyl qualifies for R&D tax relief.
I have _systematically_ made as much of my work "_re-useable_" as Open Source. 
`.some()` of it is still "Work-in-Progress" e.g: 

Our code is being used by thousands of people and has been downloaded millions of times so there is _clearly_ "_advancement_". I have spent **_many_ thousands of hours** working on the code and if I were to be _paid_ (_at my market rate_) for all that time it would _exhaust_ any revenue we have made over the past 2 years.

For a more "_precise_" figure, please consult our revenue/expenditure numbers in @iteles' spreadsheet.

Thanks!
(_unassigning myself to focus on writing more code..._ 😉 )

**Comment by markwilliamfirth** [comment]
@iteles I'm not sure the above is correct...because the amount we've spent is not equivalent to that because the remuneration will not reflect that - it has to actually come from real money that we spend. Do you know how much we've actually spent on tech ip we own? I'm unable to calculate it from the spreadsheet because I don't know what percentage of the work done that we own the ip for.

**Comment by nelsonic** [comment]
@markwilliamfirth this is _exceedingly_ simple: pay me _all_ the money I've made for the company doing Dev/QA over the last 2 years and you'll have a nice big _chunk_ out of the P&L which we can claim on R&D. ✅ 
I will _re-invest_ the money back into the business a minute later. 👍 
@iteles please handle the "_details_". thanks. ❤️

**Comment by markwilliamfirth** [comment]
@nelsonic I don't know if it's that simple as paying you all the money might drastically increase your tax

@iteles do you have an estimate for money not related to nelson's remuneration? It'll be easier if we start with that

**Comment by iteles** [comment]
@nelsonic Mark and I will take care of this, please mute notifications for this. 

@markwilliamfirth If you're able to come into dwyl tomorrow morning, let's sit and go through the open issues like this one together :+1:

**Comment by iteles** [comment]
@markwilliamfirth Let's go through this together when we have an expenditure per project from our end of year accounts https://github.com/dwyl/hq/issues/229

We'll need to think about how we deal with Nelson's contributions separately 👍

**Comment by iteles** [comment]
This has been blocked for the last 6 months, but the relevant blocker has now been moved into #373

**Comment by markwilliamfirth** [comment]
@iteles this isn't blocked - the next step is to provide estimates of what percentage of each project is R&D work, the £ figures don't block the percentage estimate

**Comment by iteles** [comment]
@markwilliamfirth As per my comment here: https://github.com/dwyl/hq/issues/112#issuecomment-286265917 this cannot be _completed_ until #373 is complete. I was just updating the issue reference seeing as you split that portion of the issue out into a new issue recently.

**Comment by markwilliamfirth** [comment]
@iteles I understand your point but I think there is a misunderstanding - the next step is to provide an estimate of a claim, to do this we need two pieces of information. One is the percentage of a project that is R&D / open source. The second is the amount of money spent on that project. Both figures can be estimated independently of each other, so there is no blocker.

**Comment by markwilliamfirth** [comment]
This is now an epic

**Next steps:**

- [ ] https://github.com/dwyl/hq/issues/391
- [ ] https://github.com/dwyl/hq/issues/229
- [ ] https://github.com/dwyl/hq/issues/339

**Comment by iteles** [comment]
Prepped and met with our R&D consultants today, great meeting.

We'll be looking to sign our contract for them to help us with this work by 14th December, have our technical onsite in early January and our claim in in late February, once #229 is finally done.

**Comment by iteles** [comment]
Met with the consultants today to ascertain which projects qualified, next step is https://github.com/dwyl/hq/issues/391

**Comment by iteles** [comment]
This issue is full of `noise`.

Closing it in favour of the clearer #438, with reference to this issue for historic purposes. https://github.com/dwyl/hq/issues/438

**Comment by assembledadam** [comment]
Randomly found this on a Google search.

May be useful to people in this thread: check out Claimer (https://www.claimer.io). It's now in public beta, and cuts the fees of specialist agencies significantly, charging 5% of what you get back instead of the 20-30% that GrantTree/Leyton charge.

Because the information gathering part is via an app, it's also a lot quicker - minutes instead of weeks. Trustpilot here: https://uk.trustpilot.com/review/claimer.io

Full disclosure: I built the app because I thought someone should end the gravy train for those specialists..! 👍

**Comment by nelsonic** [comment]
@assembledadam well done for building an app to replace over-charging human "experts" (_sales people_) who take a *chunk* out of claims for not much in the way of "service" ... 🥇 

![clamer io](https://user-images.githubusercontent.com/194400/54894666-be47cd80-4eb3-11e9-9dbd-4677807dd61b.png)

Worth noting that 5% while _much_ cheaper than the competitors is not a "flat fee".
The graph of 5% of anything is never a "flat" line; it rises in proportion to the claim.

> "_A flat fee, also referred to as a flat rate refers to a pricing structure that charges a **single fixed fee** for a service, regardless of usage._" https://en.wikipedia.org/wiki/Flat_rate

The _reason_ we used a Leyton to help us fill our R&D tax credit claim was their experience and expertise. (_basically we didn't want the **hassle** of HMRC wasting our time with questions on the claim..._) 

![image](https://user-images.githubusercontent.com/194400/54894856-ade42280-4eb4-11e9-87ca-177ad257e3c6.png)

Terms:

```
Claimer Terms of Use
Claimer is a web application which is located at https://www.claimer.io that files Research & Development tax credit claims on behalf of UK companies.

Claimer Tech Ltd is registered in England and Wales under registration number 11148907 and our principal place of business is at Claimer Tech Ltd, 17 Hurstbourne Priors, 19 Harewood Road, South Croydon, CR2 7AT ('we' or 'us').

We are registered with HMRC for Anti-Money Laundering Supervision, registration number XLML00000127874.

These Terms of Use ('Terms') apply to your ('Customer' and 'you' below) use of the website at https://www.claimer.io ('Site') and any features or services ('Services') offered through it.

To use the Services you must agree to these Terms so please read them carefully. Please note that we may change or update these Terms. We will post any changes to our Site so please revisit every so often because we will assume that you agree with them if you continue to use the Services. If you do not agree to these Terms, you must not use our Site.

1. Confidentiality - It's Your Data
As we handle sensitive information about companies, we take our legal obligations to protect that information under the Data Protection Act 1998 and EU General Data Protection Regulation ('GDPR') very seriously. To understand exactly how we handle data, check our Privacy Policy.

We want to let you know that all data you upload to the Site is 100% yours. We’ll never share or make your data or information available to anyone without your explicit permission (other than a legal requirement such as a court order).

Your data is transmitted over a secure (TLS) connection and your credentials are encrypted and hashed - we don’t store any passwords. Your company data is stored securely, and only people you’ve assigned as collaborators can access your data, per the access permissions you assigned. We expect you to ensure that your collaborators, particularly those you give Admin access to, have strong passwords, ideally different to their social network site logins.

To enhance the service, we plan to provide you with aggregated data to help you to make data-driven decisions within your claim. For example, we could provide an estimated R&D percentage of an employee. To provide such a service, the Site needs to be able to access anonymized and aggregated R&D project data across thousands of companies. You agree for specific data points from your company profile and settings to be included in such aggregated data services in non-identifiable and anonymised form, for the mutual benefit of all Claimer customers.

2. Staff Access for Customer Support Purposes
Occasionally you may contact us for online help or customer support. In such instances, for example if you have a question about a particular expense, a staff member may require temporary access via our admin systems to your account and documents in order to help you.

3. Our Obligations
We will perform the services with reasonable skill and care. The services are provided solely for the purpose set out in this Agreement. Our obligations under this Agreement are concerned with performing the services as a means to achieve a successful outcome for a R&D Tax Relief claim submitted by you, however we do not make any representations about whether or not you will receive a full or partial refund.

In the event of HMRC queries Claimer will assist you, but we accept no responsibility or liability if the R&D Tax Relief claim is rejected by HMRC for any reason.

We are not providing financial, legal, tax or any other advice. The Site should not be used as a substitute for advice from qualified legal, tax or accounting professionals.

4. Your Responsibilities
You are responsible for the accuracy and completeness of the information that you provide to us. The accuracy of the R&D Tax Relief claim prepared by Claimer will depend on the accuracy and completeness of the responses that you provide.

By instructing us, you confirm that you are authorised to act on behalf of the company for whom the Services are being sought. You must notify us as soon as you are no longer authorised to act on the company’s behalf and we will terminate this contract.

You will be responsible for reviewing the R&D Tax Relief claim report for accuracy and completeness and arrange for the claim to be filed within your tax return within 2 years of the accounting year end.

You agree to appoint us as your HMRC R&D tax agent to allow us to correspond directly with HMRC on your behalf. You also authorise us to be your nominee allowing us to enter our client bank account details on the company tax return in order to receive funds from HMRC on your behalf. Any funds received from HMRC as a result of a R&D Tax Relief Claim will be deposited into Claimer Tech client bank account, ready to be paid to you minus our fee within one working day of cleared receipt.

You will not, directly or indirectly: reverse engineer, decompile, disassemble or otherwise attempt to discover the source code, object code or underlying structure, ideas, know-how or algorithms relevant to the Services or any software, documentation or data related to the Services ('Software'); modify, translate, or create derivative works based on the Services or any Software (except to the extent expressly permitted by us or authorized within the Services); use the Services or any Software for timesharing or service bureau purposes or otherwise for the benefit of a third; or remove any proprietary notices or labels.

5. Fees
Claimer’s fee is be 5% of any R&D tax credit recovered, excluding any promotion code or discount. There is no minimum fee. There will also be no fee in the unlikely event that the claim is unsuccessful.

If the claim results in a payable tax credit, we will collect this payment on behalf of you. On cleared receipt of the tax credit payment from HMRC, we will transfer this sum minus our fee to your nominated bank account within 1 working day. We will follow up this payment with an invoice and receipt of payment.

If the claim is successful but does not result in a payable tax credit (rather that you elect to carry forward or back losses), we will send you an invoice for our fee, with payment due within 30 days.

Fees are exclusive of VAT.

6. Value Added Tax
Claimer Tech Ltd is not yet registered for VAT, and so will not be charged VAT on fees. We will notify you ahead of time if this changes.

7. Payment of Invoices
Fees are due on the delivery of our invoice without any deduction by way of set off, counterclaim or otherwise. We reserve the right to charge interest from the date of invoice on the outstanding balance of any invoice unpaid 30 days after its date at the rate specified in the Late Payment of Commercial Debts (Interest) Act 1998.

We reserve the right to recover, on a full indemnity basis, any costs incurred by us in collecting overdue payments.

8. Termination
You may terminate your instructions at any time (subject to these Terms of Use) on notice in writing. We may also withdraw from acting for you, but normally will only do so for professional reasons or arising from your failure to comply with these Terms of Use.

We will render an invoice to you which will become payable in accordance with these Terms of Use for all work carried out up to or in consequence of the conclusion of our contract with you.

9. Critical Dates
Once your claim is completed, we cannot accept an ongoing responsibility for reminding you of critical dates relating to matters such as R&D claim or corporation tax filing deadlines unless we have current and specific written instructions from you to deal with them prior to the critical date.

10. Copyright and Other Intellectual Property Rights
Unless otherwise specifically agreed in writing, we retain the copyright and other intellectual property rights in all written or other materials supplied to you. If materials prepared by us are passed on to or are disclosed by you to third parties, then you are liable for the payment of proper professional charges for their use and all expenses or losses incurred in enforcing our intellectual property rights.

11. Warranty and Disclaimer
We will use reasonable efforts consistent with prevailing industry standards to maintain the Services in a manner which minimizes errors and interruptions in the Services and will perform the Implementation Services in a professional and workmanlike manner. Services may be temporarily unavailable for scheduled maintenance or for unscheduled emergency maintenance, either by us or by third-party providers, or because of other causes beyond our reasonable control, but we will use reasonable efforts to provide advance notice in writing or by e-mail of any scheduled service disruption.

However, we do not warrant that the Services will be uninterrupted or error free; nor do we make any warranty as to the results that may be obtained from use of the Services.

EXCEPT AS EXPRESSLY SET FORTH IN THIS SECTION, THE SERVICES AND IMPLEMENTATION SERVICES ARE PROVIDED 'AS IS' AND WE DISCLAIM ALL WARRANTIES, EXPRESS OR IMPLIED, INCLUDING, BUT NOT LIMITED TO, IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT.

12. Indemnity
We will hold you harmless from liability to third parties resulting from infringement by the Service of any UK patent or any copyright or misappropriation of any trade secret, provided we are promptly notified of any and all threats, claims and proceedings related thereto and given reasonable assistance and the opportunity to assume sole control over defense and settlement; We will not be responsible for any settlement it does not approve in writing.

The foregoing obligations do not apply with respect to portions or components of the Service (i) not supplied by us, (ii) made in whole or in part in accordance with your specifications, (iii) that are modified after delivery by us, (iv) combined with other products, processes or materials where the alleged infringement relates to such combination, (v) where we continue allegedly infringing activity after being notified thereof or after being informed of modifications that would have avoided the alleged infringement, or (vi) where your use of the Service is not strictly in accordance with these Terms.

If, due to a claim of infringement, the Services are held by a court of competent jurisdiction to be or are believed by us to be infringing, we may, at our option and expense (a) replace or modify the Service to be non-infringing provided that such modification or replacement contains substantially similar features and functionality, (b) obtain for you a license to continue using the Service, or (c) if neither of the foregoing is commercially practicable, terminate our agreement under these Terms and your rights hereunder and provide you a refund of any prepaid, unused fees for the Service.

13. Limitation of Liability
NOTWITHSTANDING ANYTHING TO THE CONTRARY, EXCEPT FOR BODILY INJURY OF A PERSON, WE AND OUR OFFICERS, AFFILIATES, REPRESENTATIVES, CONTRACTORS AND EMPLOYEES WILL NOT BE RESPONSIBLE OR LIABLE WITH RESPECT TO ANY SUBJECT MATTER OF THESE TERMS RELATED THERETO UNDER ANY CONTRACT, NEGLIGENCE, STRICT LIABILITY OR OTHER THEORY: (A) FOR ERROR OR INTERRUPTION OF USE OR FOR LOSS OR INACCURACY OR CORRUPTION OF DATA OR COST OF PROCUREMENT OF SUBSTITUTE GOODS, SERVICES OR TECHNOLOGY OR LOSS OF BUSINESS; (B) FOR ANY INDIRECT, EXEMPLARY, INCIDENTAL, SPECIAL OR CONSEQUENTIAL DAMAGES; (C) FOR ANY MATTER BEYOND OUR REASONABLE CONTROL; OR (D) FOR ANY AMOUNTS THAT, TOGETHER WITH AMOUNTS ASSOCIATED WITH ALL OTHER CLAIMS, EXCEED THE FEES PAID BY YOU TO US FOR THE SERVICES UNDER THESE TERMS IN THE 12 MONTHS PRIOR TO THE ACT THAT GAVE RISE TO THE LIABILITY, IN EACH CASE, WHETHER OR NOT WE HAVE BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.

14. Miscellaneous
If any reference in these Terms is found to be unenforceable or invalid, that provision will be limited or eliminated to the minimum extent necessary so that these Terms will otherwise remain in full force and effect and enforceable. Our agreement under these Terms is not assignable, transferable or sublicensable by you except with our prior written consent. We may transfer and assign any of our rights and obligations under these Terms without consent. Our agreement under these Terms is the complete and exclusive statement of the mutual understanding of the parties and supersedes and cancels all previous written and oral agreements, communications and other understandings relating to the subject matter of these Terms, and that all waivers and modifications must be in a writing signed by both parties, except as otherwise provided herein. No agency, partnership, joint venture, or employment is created as a result of our agreement under these Terms and you do not have any authority of any kind to bind us in any respect whatsoever. In any action or proceeding to enforce rights under these Terms, the prevailing party will be entitled to recover costs and legal fees.

All notices under these Terms will be in writing and will be deemed to have been duly given when received, or if transmitted by email, the day after it is sent.

Our agreement under these Terms will be governed by the laws of England and Wales and we both agree to the exclusive jurisdiction of the courts of England and Wales.

15. How to Contact Us
If you have a question about these Terms of Use or a complaint, please email us at support@claimer.io and we will respond as promptly as we can, or in the time period prescribed by law, whichever is sooner.
```

![image](https://user-images.githubusercontent.com/194400/54895035-7c1f8b80-4eb5-11e9-8065-d55b22952dcf.png)

Guessing the part of verifying company details hasn't been automated yet. 
Still, looks promising. 👍

**Comment by assembledadam** [comment]
@nelsonic Hey Nelson.

Thanks for the detailed reply. Below is my response to your points one by one.

> Worth noting that 5% while much cheaper than the competitors is not a "flat fee".
> The graph of 5% of anything is never a "flat" line; it rises in proportion to the claim.

Good spot, just a grammar fail there, now fixed. As I mentioned the app is in public beta - the homepage generally sucks in a few ways. There's a vastly improved one coming very soon (< 2 weeks), which better explains the service.

Also worth noting, which will be illustrated in the new homepage, that Claimer's 5% fee is capped at £5,000 (i.e. no fee beyond a claim size of £100k).

In addition, we offer a *fixed* fee of £250, including submission, if you only wish to use the software to produce your own claim - i.e. without our R&D experts assessing and producing the claim for you.

> The reason we used a Leyton to help us fill our R&D tax credit claim was their experience and expertise. (basically we didn't want the hassle of HMRC wasting our time with questions on the claim...)

Understood, but Claimer also utilises experienced R&D experts to produce your claim, in fact to a larger extent than Leyton and others:

The app allows you to enter your information more efficiently than with traditional specialists like Leyton, in a 5 step process. It connects to your accounting software (currently supported: Xero, coming soon: FreeAgent, Sage, Quickbooks, and Wave) which also significantly speeds up the claim.

On submission, the claim goes an R&D expert that checks the company and costs for validity, ensures the claim is maximised, and then they write the report using the information you have provided. We have processed over 35 claims now since January with a 100% success rate, and no queries from HMRC.

In addition all other agencies I have come across (such as Leyton, GrantTree, ForrestBrown, etc) have accountants write your R&D technical narrative. These accountants do not have more than a superficial understanding of the technology and development undertaken, which often results in flawed (invalid) technical reports according to the legislation. I have seen them - and they can be awful. HMRC does not currently check every claim, which can result in a false sense of confidence.

With Claimer, the R&D expert that writes your report is also an expert in the field of your R&D claim (e.g. software, hardware, manufacturing), so the claim is always technically accurate and within the R&D guidelines. This explains why we have not had a single query from HMRC.

PS. probably unnecessary to paste the entire terms and conditions 😶

> Guessing the part of verifying company details hasn't been automated yet.

The company is a registered HMRC agent, and is also supervised by HMRC. As such, by law we must carry out anti-money laundering checks, as well as several manual checks to ensure the public records of the company indicate that it qualifies for R&D tax relief. Would be a lot of work to automate that at the moment!

If you'd consider using the service I'd be happy to demo it to you on a call - just pop me a message (adam@claimer.io).
